packages/components/src/InputGroup/InputGroup.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from "react";
+import classnames from "classnames";
+import styles from "./InputGroup.css";
+
+interface InputGroupProps {
+  readonly children: ReactElement | ReactElement[];
+  readonly flowDirection?: "horizontal" | "vertical";
+}
+
+export function InputGroup({
+  children,
+  flowDirection = "vertical",
+}: InputGroupProps) {
+  if (checkChildren(children)) return <></>;
+
+  const className = classnames(styles.inputGroup, styles[flowDirection]);
+  return <div className={className}>{children}</div>;
+}
+
+function checkChildren(childs: ReactElement | ReactElement[]): boolean {
```

Comment:
Name suggestions?

```suggestion
function invalidGroupNesting(children: ReactElement | ReactElement[]): boolean {
```

Reply:
smh - meant to improve this. Good suggestion, thanks.